lib/index.js

```diff
@@ -98,7 +98,16 @@ export default {
           return null;
         }
 
-        const fileText = editor.getText();
+        const removePHP = str => str.replace(/<\?php(?:[\s\S])+?\?>/gi, (match) => {
```

Comment:
You need to handle short open tags for starters, and I'm quite suspicious that this would work at all to properly parse out the PHP sections of the document, after all... [ZA̡͊͠͝LGΌ ISͮ̂҉̯͈͕̹̘̱ TO͇̹̺ͅƝ̴ȳ̳ TH̘Ë͖́̉ ͠P̯͍̭O̚​N̐Y̡ H̸̡̪̯ͨ͊̽̅̾̎Ȩ̬̩̾͛ͪ̈́̀́͘ ̶̧̨̱̹̭̯ͧ̾ͬC̷̙̲̝͖ͭ̏ͥͮ͟Oͮ͏̮̪̝͍M̲̖͊̒ͪͩͬ̚̚͜Ȇ̴̟̟͙̞ͩ͌͝S̨̥̫͎̭ͯ̿̔̀ͅ](https://stackoverflow.com/a/1732454/631338).

Reply:
This is working fine with `<?php ?>` tags, I'll amend to allow short tags